portfolio/src/main/webapp/script.js

```diff
@@ -127,4 +127,51 @@ function createListElement(comment) {
 // Deletes all comments from datastore and refreshes.
 function clearComments() {
   fetch("/delete-data", {method: 'POST'}).then(() => getComments());
+}
+
+// Refresh page causes the comments and login status to refresh.
+function refresh() {
+  getComments();
```

Comment:
I don't see getComments() defined anywhere. Is it just me not finding it in the GitHub UI?